downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -111,6 +111,42 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### Improvements
 
+- With this update, resources have been moved to the package level and module-level resources are deprecated. Resources available in module resource paths will be ignored. To be recognized as valid, any existing module-level resources must be relocated to the resources directory in the package root.
+  Any resources within the current package, as well as those exported from package dependencies, can now be accessed through an interop function.
```

Comment:
Do we need this? Should the user write this? If so, not sure if we should emphasize it in the release note.

Reply:
Added this as a note with 2bcdfaa